x-pack/plugins/fleet/public/components/package_policy_actions_menu.tsx

```diff
@@ -92,23 +92,13 @@ export const PackagePolicyActionsMenu: React.FunctionComponent<{
       : []),
     <EuiContextMenuItem
       data-test-subj="PackagePolicyActionsEditItem"
-      disabled={
-        !canWriteIntegrationPolicies || !agentPolicy || (agentPolicy?.supports_agentless ?? false)
-      }
+      disabled={!canWriteIntegrationPolicies || !agentPolicy}
```

Comment:
@juliaElastic As a product requirement, we would ensure Agentless Integration's edit feature is enabled. The use  case here,  we  would like user  to change input configurations. However, in edit mode  we did will disable **SetupTechnology Selector** to prevent user from switching back from Agentless to Agent-based.

Reply:
There is one more place the Edit action was disabled with a tooltip, it should be removed for consistency: https://github.com/elastic/kibana/pull/189612/files#diff-401ad2aa19dd4a96823444957e03c9d7c8d8e4b6ca31ad8f155fc73f0e8dde77

I think it makes sense to allow editing the integration policy config, and only disable changing the Setup technology selector to be changed on edit. FYI @kpollich as this changes the implementation we did in https://github.com/elastic/kibana/issues/183863

Reply:
Nice catch! Thank you!